elife/scripts/rds-perms2.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
to be fair I'll probably just call it `psql-app-db-perms.sh` or something super inventive like that.